docs/DeveloperGuide.adoc

```diff
@@ -279,51 +287,88 @@ Refer to the guide <<DevOps#, here>>.
+
+|`* * *` |Admin In-Charge |delete an Entity by name |remove entries I no longer need
+
+|`* * *` |Admin In-Charge |add an Entity by name and contact information |update the list of Entities
+
+|`* * *` |Admin In-Charge |updated an Entity by name and contact information |update the specific entries
+
+|`* * *` |Admin In-Charge |register individuals en-masse(with provided registration information) | avoid tedious manual registration
+
+|`* * *` |Admin In-Charge |keep track of winning teams and the prizes won | ensure that the prize-giving ceremony runs smoothly
+
+|`* * *` |Admin In-Charge |keep track of winning teams and the prizes won | ensure that the prize-giving ceremony runs smoothly
+
+|`* * *` |Admin In-Charge |make sure that I will be notified on any wrong commands that I type | make sure that I do not accidentally clutter up my list of entries
+
+|`* * *` |Admin In-Charge |have a readable and organised User Interface | understand the output of my commands
+
+|`* * *` |Admin In-Charge |keep track of participants who signed up late or after the event has filled up into a waitlist | manage them in case available space turns up during the Event
+
+|`* * *` |Admin In-Charge |manually match Teams to Mentor | know which Mentor is in charge of a team
+
+|`* * *` |Admin In-Charge |keep track of where each Team or Mentor is seating|usher them to their places during the actual event
+
+|`* *` |Admin In-Charge |know my sponsor's needs and arrival time | adequately cater to their needs and allocate manpower accordingly
+
+|`* *` |Admin In-Charge |keep track of inventory of swag | make sure they are adequately catered to all participants
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* *` |Admin In-Charge |keep track of amount of food or catering | make sure they are adequately catered to all participants
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`*` |Admin In-Charge |automatically match Teams to Mentor by their expertise and project type of the Team | do not need to perform the matching manually
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`*` |Admin In-Charge |schedule meetings between Teams and Mentors | lets Mentors know when to consult each Team in an organised manner
 |=======================================================================
 
 _{More to be added}_
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `HackathonManager` and the *Actor* is the `user`, unless specified otherwise)
 
 [discrete]
```

Comment:
This should be numbered together with the rest of Use Cases and should not be discrete.